src/buffer/out/Row.cpp

```diff
@@ -969,26 +1009,7 @@ til::CoordType ROW::MeasureRight() const noexcept
         return width;
     }
 
-    const auto text = GetText();
-    const auto beg = text.begin();
-    const auto end = text.end();
-    auto it = end;
-
-    for (; it != beg; --it)
-    {
-        // it[-1] is safe as `it` is always greater than `beg` (loop invariant).
-        if (til::at(it, -1) != L' ')
-        {
-            break;
-        }
-    }
-
-    // We're supposed to return the measurement in cells and not characters
-    // and therefore simply calculating `it - beg` would be wrong.
-    //
-    // An example: The row is 10 cells wide and `it` points to the second character.
-    // `it - beg` would return 1, but it's possible it's actually 1 wide glyph and 8 whitespace.
-    return gsl::narrow_cast<til::CoordType>(_columnCount - (end - it));

```

Comment:
`_columnCount` should be `GetReadableColumnCount()`?

Using `_columnCount` directly doesn't work for `LineRendition != SingleWidth`

Reply:
Oh... right, I completely forgot about fixing this bug! Thank you for finding and fixing it. 🙂